examples/components/bottomSheets/bottomSheets.jsx

```diff
@@ -35,4 +35,4 @@ BottomSheets.defaultProps = {
   className: '',
 }
 
-export default router('BOTTOMSHEETS')(onlyUpdateForPropTypes(BottomSheets))
+export default forRoute('BOTTOMSHEETS', { getState: state => state.router })(onlyUpdateForPropTypes(BottomSheets))
```

Comment:
Why do you need to specify the getState here if you import the forRoute from your helper?

Reply:
it was a test for this component, I forgot to rollback